simple-wp-helpdesk/src/Rest/TicketController.php

```
@@ -0,0 +1,363 @@
+				'comment_content'      => $body,
+				'comment_approved'     => 1,
+			)
+		);
+
+		if ( $internal && $comment_id ) {
+			update_comment_meta( $comment_id, '_is_internal_note', '1' );
+		}
+
+		// REST path inserts directly; bump explicitly (B2 hooks cover PHP-side
+		// sites, not this REST route).
+		TicketVersion::bump( $id );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\SWH\Concurrency\TicketVersion' in method 'create_reply'. (undefined)

(StaticAccess)